README.md

```diff
@@ -1 +1,45 @@
-# Israel-Portfolio
\ No newline at end of file
+# Israel-Portfolio
+this is an exercise Project designed to create step by step our portfolio.
+
+
+# Project Built With
+HTML
+CSS
+
+# Getting Started
+Please follow the instructions to clone my repository in your computer
+
+To get a local copy, follow these simple example steps.
+
+Clone this repository or download the Zip folder:
+
+git clone https://github.com/chiztechnology/Israel-Portfolio
+
+Navigate to the location of the folder in your machine:
+
+you@your-Pc-name:~$ cd <folder>
+Press Enter to navigate to your local clone.
```

Comment:
- [ **OPTIONAL** ] I think it would be better you wrap all you command lines inside backsticks(**`**) to make them look like command lines and different other normal text